spec/tool-bar-spec.js

```diff
@@ -95,7 +95,13 @@ describe('Tool Bar package', () => {
           tooltip: 'About Atom'
         });
         expect(toolBar.children.length).toBe(1);
-        expect(toolBar.firstChild.dataset.originalTitle).toBe('About Atom');
+        const element = toolBar.firstChild;
```

Comment:
The way of testing those is based on atom: it's not super pretty but works
https://github.com/atom/atom/blob/master/spec/tooltip-manager-spec.coffee